src/marketing/type/typography.scss

```diff
@@ -113,6 +111,18 @@
   }
 }
 
-.text-medium {
-  font-weight: $font-weight-medium !important;
+.text-ultra-light {
```

Comment:
Should this be `.text-extra-light` to match `.text-extra-bold`?

Reply:
@simurai I think that makes sense! ✨ changing.

Reply:
@simurai Updated! ✨

Reply:
👍  With the utility name change, we might also want to change the variable [`$font-weight-ultra-light`](https://github.com/primer/css/pull/2192/files#diff-cb94cf5afa0ea8c52869cc8fcbf1c89736a5c69629624347f62b924da09ac590R8) to `$font-weight-extra-light`?

Reply:
Yesss good point 😅